--- a/CON80/SM2	2026-09-11 05:37:50.903432593 -0500
+++ b/CON80/SM2	2026-09-11 05:37:50.921432736 -0500
@@ -150,6 +150,32 @@
          INSERT  #DPGLDIS                                           SM2 008826DJ
          INSERT  #Y157001                                           SM2 009200DC
 *STACK *                           SM2 STACKS                       SM2 009300CU
+         STACK   $0REXRMS                                           SM2 009400CU
+         STACK   $0RPORPO                                           SM2 009450CU
+         STACK   $0RMCSPE                                           SM2 009500CU
+         STACK   $0RMDSPE                                           SM2 009550CU
+         STACK   $0SAFACQ                                           SM2 009600CU
+         STACK   $0SLSSPE                                           SM2 009650CU
+         STACK   $0SM2OPS                                           SM2 009700CU
+         STACK   $0SPNINT                                           SM2 009750CU
+         STACK   $0SPSPSP                                           SM2 009800CU
+         STACK   $0SSPEXE                                           SM2 009850CU
+         STACK   $0STCCYC                                           SM2 009900CU
+         STACK   $0SCKPNT                                           SM2 009950CU
+         STACK   $0STSSPE                                           SM2 010000CU
+         STACK   $0SULUPL                                           SM2 010050CU
+         STACK   $0SFLFOR                                           SM2 010100CU
+         STACK   $0DMPMMM                                           SM2 010150CU
+         STACK   $0SCOSPE                                           SM2 010200CU
+         STACK   $0DGMWRT                                           SM2 010250CU
+         STACK   $0VMMMMS                                           SM2 010300CU
+         STACK   $0PC1SPE                                           SM2 010350CU
+         STACK   $0PC2SPE                                           SM2 010400CU
+         STACK   $0PGDATA                                           SM2 010450CU
+         STACK   $0PGASS1                                           SM2 010500CU
+         STACK   $0PGBSS2                                           SM2 010550CU
+         STACK   $0PGCSS3                                           SM2 010600CU
+         STACK   $0PGMMMR                                           SM2 010650CU
 *********                                                           SM2 011200CU
          INCLUDE CONCARDS(SM2TAB)                                   SM2 011300CU
          INCLUDE CONCARDS(SM2MSPS)                                  SM2 011400CU
